### ISSUE TITLE
blind screen reader user unable to access hotkey function

### ISSUE THREAD

**Issue by whocrazy** [clsid2/mpc-hc#2751]
I am trying to assign a new hotkey, I use a screen reader called NVDA, non visual desktop access.  When I attempt to set the new hotkey, the keyboard freezes, and I can't use NVDA to navigate so that I can figure out what to do in order to assign the new key.  I would really love this to be fixed if possible.

**Comment by clsid2** [comment]
Click inside field.
Type combo.
Click outside field to apply, or click on button at end of field and select Apply in context menu.

There will be no changes and I am not interested in any compatibility with screen readers.

**Comment by whocrazy** [comment]
why on earth not?

**Comment by clsid2** [comment]
I do this for free in my spare time.

For $1000 per hour I do whatever you want.

**Comment by whocrazy** [comment]
How is my issue different than all the other bugs and issues that every one else has reported on here? or are you planning on charging $1000 an hour to fix all of those too? for the most part, Media player classic works extremely well with screen readers, there aren't any huge overhauls that must be made to media player classic, no proprietary software or plugins that need to be installed, no special screen layout or anything like that. Just a way to stop the keyboard from being frozen during the hotkey setup, that's the only thing that needs to be changed, everything else can be left as is.
